### PR TITLE
SegmentEngine 1.0.0

### PR DESCRIPTION
**Pull request by nnlamb25**
@weshenderson for review

### CODE REVIEW

**Comment by nnlamb25** [comment]
For [#3197](https://github.com/Factual/engine/issues/3197)

File: library/SegmentEngineUserJourneyHandler.m [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  
+  // Geographies information
+  Geographies *geographies = currentPlace.geographies;
+  NSString *country = geographies.country;
+  NSArray<NSString*> *localities = geographies.localities;
+  NSString *postcode = geographies.postcode;
+  NSString *region = geographies.region;
+  
+  // Get duration of span
+  NSNumber *duration = !startTimestampUnavailable && !endTimestampUnavailable ? @(endTimestamp - startTimestamp) : 0;
```
Comment (lispingit) [inline library/SegmentEngineUserJourneyHandler.m]: The `0` case will result in `nil` rather than `NSNumber` here.

File: example/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+**(1)** Locate your Factual Engine API Key from the [Factual Engine Dashboard](https://engine.factual.com/garage)
+
+![Dashboard image](./images/apikey.png)
+
+**(2)** Add your Factual Location Engine API Key to [Configuration.m](https://github.com/Factual/engine-airship-integration-ios/blob/master/example/UrbanAirshipEngineExample/Configuration.m#L15)
+
+**(3)** Locate your Segment API Key for your app from the [Segment Dashboard](https://dashboard.braze.com).  In **Sources** select your app, open the **settings** tab, and select **API Keys**.
+
+**(4)** Add your Braze SDK API Key to [Configuration.m](https://github.com/Factual/engine-airship-integration-ios/blob/master/example/UrbanAirshipEngineExample/Configuration.m#L19)
```
Comment (lispingit) [inline example/README.md]: These are Airship links rather than Segment.

**Review by lispingit — CHANGES_REQUESTED** [review]
Looks good -- only a couple minor issues that I found. That said, given the size of this change, make sure this is throughly tested.

**Review by lispingit — APPROVED** [review]

**Pull request merged**